app/views/unattended/provisioning_templates/snippet/first_boot_setup_script_non_systemd.erb

```diff
@@ -0,0 +1,10 @@
+<%#
+kind: snippet
+name: first_boot_setup_script_non_systemd
+model: ProvisioningTemplate
+snippet: true
+description: |
+  First boot setup script for non-systemd systems.
+-%>
+<%= indent(2, skip1: true) { snippet('built', :variables => { :endpoint => 'built', :method => 'POST', :body_file => '/root/install.post.log' }) } -%>
+mv /tmp/first_boot_setup_script.sh /tmp/first_boot_setup_script.disabled
```

Comment:
Do we need this as a separate snippet? IMHO this could be part of first_boot_setup_non_systemd.erb.